src/pages/SchedulePage.tsx

```diff
@@ -121,8 +126,8 @@ const SchedulePage: React.FC<SchedulePageProps> = ({
               value={segment}
               onIonChange={(e) => setSegment(e.detail.value as any)}
             >
-              <IonSegmentButton value="all">All</IonSegmentButton>
-              <IonSegmentButton value="favorites">Favorites</IonSegmentButton>
+              <IonSegmentButton value="all">{intl.formatMessage({id: "schedule.all"})}</IonSegmentButton>
+              <IonSegmentButton value="favorites">{intl.formatMessage({id: "schedule.favorites"})}</IonSegmentButton>
```

Comment:
Please use on this particular line react-intl `<FormattedMessage` because it can be used as child of `<IonSegmentButton>`. It is very nice to demonstrate also react-intl hook `int.formatMessage` but please find proper place in the codebase for example `<img alt={intl.formatMessage()}` or `<IonToast message` or any other with human readable string in react component property.